### PR TITLE
Built-in Authorizing Payment scene

### PR DESCRIPTION
**Pull request by pitiphong-p**
Since the Authorizing Payment is a common operation when using Omise service now. I think it would be great if we have the built-in way to support the Authorizing Payment in the SDK.

This PR introduces the `AuthorizingPaymentActivity` scene which replaces the old `Verify3DSActivity`. This new OmiseAuthorizingPaymentViewController requires the expected return URL patterns in order to make it work properly. The merchants/library-users need to specify their return URL patterns to this scene.

### CODE REVIEW

**Comment by chakrit** [comment]
Looks like we need to bump minor version with this.

**Comment by pitiphong-p** [comment]
Yeah, will do that after this PR got merged

File: .idea/misc.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <ConfirmationsSetting value="0" id="Add" />
     <ConfirmationsSetting value="0" id="Remove" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (chakrit) [inline .idea/misc.xml]: @pitiphong-p 🤔 intentional?

I think we need to stick to Java 7 na.

File: sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        }
+
+        expectedReturnURLPatterns = new Uri[returnURLPatternList.size()];
+        expectedReturnURLPatterns = returnURLPatternList.toArray(expectedReturnURLPatterns);
+
+        if (authorizedURL != null && authorizedURL.toString() != null) {
+            webView.loadUrl(authorizedURL.toString());
+        }
+
+        webView.setWebViewClient(new WebViewClient() {
```
Comment (chakrit) [inline sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.java]: @pitiphong-p Can we split this into an inner class inside this class? I think it'll be more readable on its own unit.

File: .idea/misc.xml (hunk 1 above)
Comment (pitiphong-p) [inline .idea/misc.xml]: Nope, OK will try to find how to revert that

File: sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.java (hunk 2 above)
Comment (pitiphong-p) [inline sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.java]: Ummm 🤔, sounds interesting. OK will try to do that

**Comment by pitiphong-p** [comment]
Updated with the inner class `AuthorizingPaymentURLVerifier`

**Pull request merged**